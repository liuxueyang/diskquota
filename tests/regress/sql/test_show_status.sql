select * from diskquota.status();

select from diskquota.enable_hardlimit();
select * from diskquota.status();

select from diskquota.disable_hardlimit();
select * from diskquota.status();

select from diskquota.pause();
select * from diskquota.status();

select from diskquota.enable_hardlimit();
select * from diskquota.status();

select from diskquota.disable_hardlimit();
select * from diskquota.status();

select from diskquota.resume();
select from diskquota.disable_hardlimit();
select * from diskquota.status();
