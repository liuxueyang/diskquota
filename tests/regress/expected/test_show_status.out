select * from diskquota.status();
    name     |  status  
-------------+----------
 soft limits | enabled
 hard limits | disabled
(2 rows)

select from diskquota.enable_hardlimit();
--
(1 row)

select * from diskquota.status();
    name     | status  
-------------+---------
 soft limits | enabled
 hard limits | enabled
(2 rows)

select from diskquota.disable_hardlimit();
--
(1 row)

select * from diskquota.status();
    name     |  status  
-------------+----------
 soft limits | enabled
 hard limits | disabled
(2 rows)

select from diskquota.pause();
--
(1 row)

select * from diskquota.status();
    name     |  status  
-------------+----------
 soft limits | paused
 hard limits | disabled
(2 rows)

select from diskquota.enable_hardlimit();
--
(1 row)

select * from diskquota.status();
    name     | status 
-------------+--------
 soft limits | paused
 hard limits | paused
(2 rows)

select from diskquota.disable_hardlimit();
--
(1 row)

select * from diskquota.status();
    name     |  status  
-------------+----------
 soft limits | paused
 hard limits | disabled
(2 rows)

select from diskquota.resume();
--
(1 row)

select from diskquota.disable_hardlimit();
--
(1 row)

select * from diskquota.status();
    name     |  status  
-------------+----------
 soft limits | enabled
 hard limits | disabled
(2 rows)

